taipy/core/data/aws_s3.py

```diff
@@ -86,17 +86,7 @@ def __init__(
         self,
         config_id: str,
         scope: Scope,
-        id: Optional[DataNodeId] = None,
-        owner_id: Optional[str] = None,
-        parent_ids: Optional[Set[str]] = None,
-        last_edit_date: Optional[datetime] = None,
-        edits: Optional[List[Edit]] = None,
-        version: str = None,
-        validity_period: Optional[timedelta] = None,
-        edit_in_progress: bool = False,
-        editor_id: Optional[str] = None,
-        editor_expiration_date: Optional[datetime] = None,
-        properties: Optional[Dict] = None,
+        **properties: Any,
```

Comment:
Why are these properties removed?

FYI, these are properties that are required for the DataNode entity